build/azure-pipeline/workflow-pr.yml

```diff
@@ -105,6 +105,13 @@ stages:
           #
           # Install build tools
           #
+          - script: |
+              /bin/bash -c "$(curl -fSs https://get-nats.io/install.sh)"
+            displayName: Install NATS for Linux/MacOS
+            condition: and(succeeded(), ne(variables['target_os'], 'windows'))
```

Comment:
If Nats current do not support Windoes, we need to document it somewhere. I am not sure where to document it, maybe in a README of the messaing crate.

Reply:
~added Windows support ✅